### PR TITLE
Implemented basic Symfony commands

### PR DESCRIPTION
**Pull request by JanTvrdik**

### CODE REVIEW

File: src/Bridges/SymfonyConsole/CreateCommand.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	}
+
+
+	/**
+	 * @param  string $label
+	 * @return string
+	 */
+	private function getFileName($label)
+	{
+		return date('Y-m-d-His-') . Strings::webalize($label, '.') . '.sql';
```
Comment (hrach) [inline src/Bridges/SymfonyConsole/CreateCommand.php]: His? we used basic counter.

Comment (JanTvrdik) [inline src/Bridges/SymfonyConsole/CreateCommand.php]: counter does not work well we multiple branches and is harder to implement.

File: src/Bridges/NetteDI/MigrationsExtension.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		} elseif ($dbal instanceof Nette\DI\Statement) {
+			return Nette\DI\Compiler::filterArguments([$dbal])[0];
+
+		} elseif (is_string($dbal) && isset($this->dbals[$dbal])) {
+			return new Nette\DI\Statement($this->dbals[$dbal]);
+
+		} else {
+			throw new Nette\InvalidArgumentException();
+		}
+	}
```
Comment (hrach) [inline src/Bridges/NetteDI/MigrationsExtension.php]: empty line

File: src/Bridges/NetteDI/MigrationsExtension.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+		} elseif ($driver instanceof Nette\DI\Statement) {
+			return Nette\DI\Compiler::filterArguments([$driver])[0];
+
+		} elseif (is_string($driver) && isset($this->drivers[$driver])) {
+			$dbal = $this->getDbal($dbal);
+			return new Nette\DI\Statement($this->drivers[$driver], [$dbal]);
+
+		} else {
+			throw new Nette\InvalidArgumentException();
```
Comment (hrach) [inline src/Bridges/NetteDI/MigrationsExtension.php]: Seems like wrong exception.

File: src/Bridges/SymfonyConsole/BaseCommand.php [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+use Nextras\Migrations\Entities\Group;
+use Nextras\Migrations\Extensions;
+use Nextras\Migrations\IDriver;
+use Nextras\Migrations\Printers\Console;
+use Symfony\Component\Console\Command\Command;
+
+
+abstract class BaseCommand extends Command
+{
+
```
Comment (hrach) [inline src/Bridges/SymfonyConsole/BaseCommand.php]: empty line

File: src/Bridges/SymfonyConsole/BaseCommand.php [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+		$basicData = new Group();
+		$basicData->enabled = TRUE;
+		$basicData->name = 'basic-data';
+		$basicData->directory = $this->dir . '/basic-data';
+		$basicData->dependencies = ['structures'];
+
+		$dummyData = new Group();
+		$dummyData->enabled = $withDummmy;
+		$dummyData->name = 'dummy-data';
```
Comment (hrach) [inline src/Bridges/SymfonyConsole/BaseCommand.php]: I'm still not sure with "dummy".

File: src/Bridges/SymfonyConsole/BaseCommand.php [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+	/**
+	 * @return array (extension => IExtensionHandler)
+	 */
+	protected function getExtensionHandlers()
+	{
+		return [
+			'sql' => new Extensions\SqlHandler($this->driver),
+			'php' => new Extensions\PhpHandler($this->phpParams),
+		];
+	}
```
Comment (hrach) [inline src/Bridges/SymfonyConsole/BaseCommand.php]: empty line

File: src/Bridges/SymfonyConsole/CreateCommand.php [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+	 */
+	private function getDirectory($type)
+	{
+		foreach ($this->getGroups(TRUE) as $group) {
+			if (Strings::startsWith($group->name, $type)) {
+				return $group->directory;
+			}
+		}
+
+		throw new \InvalidArgumentException("Unknown type '$type' given, expected on of 's', 'b' or 'd'.");
```
Comment (hrach) [inline src/Bridges/SymfonyConsole/CreateCommand.php]: wrong exception

File: src/Bridges/NetteDI/MigrationsExtension.php (hunk 2 above)
Comment (hrach) [inline src/Bridges/NetteDI/MigrationsExtension.php]: empty line

**Comment by JanTvrdik** [comment]
@hrach merge?

**Pull request merged**